lib/WebService/Async/Segment.pm

```diff
@@ -0,0 +1,245 @@
+package WebService::Async::Segment;
+
+use strict;
+use warnings;
+
+use Net::Async::HTTP;
+use IO::Async::Loop;
+use Scalar::Util qw(blessed);
+use URI::Template;
+use JSON::MaybeUTF8 qw(encode_json_utf8 decode_json_utf8);
+use Syntax::Keyword::Try;
+use Log::Any qw($log);
+#use Date::Utility;
```

Comment:
This line should be dropped.

Reply:
Excuse me. It was commented my mistake. It is actually used in lines below.